src/SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests/Services/MockOuterApi.cs

```diff
@@ -1,25 +1,46 @@
 ﻿using System;
-using WireMock.RequestBuilders;
-using WireMock.ResponseBuilders;
 using WireMock.Server;
 
 namespace SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests.Services
 {
     public class MockOuterApi : IDisposable
     {
+        private bool _isDisposed;
+
         public WireMockServer MockServer { get; }
-        public Uri BaseAddress { get; }
+        public string BaseAddress { get; }
 
         public MockOuterApi()
         {
             MockServer = WireMockServer.Start(ssl: false);
-            BaseAddress = new Uri(MockServer.Urls[0] + "/api");
+            BaseAddress = MockServer.Urls[0];
         }
 
         public void Dispose()
         {
-            MockServer?.Stop();
-            MockServer?.Dispose();
+            Dispose(true);
```

Comment:
I don't think it's necessary to use this pattern here, we are not inheriting from `MockOuterApi`, and this just adds code and complexity without benefit.

Reply:
My only comment would be the pattern is pretty standard (and is seen in most of our dispose patterns) in our software. So it is easier just to copy that style. The problem before was addressing was, there  was a Dispose pattern which wasn't implemented correctly, so this was to address that.

Reply:
I am not a big believer copying patterns without good reason, and to me it just seems more code for the sake of it rather than adding any benefit.  In fact I'd say that this pattern is actually harming the codebase as it adds extra complexity that is completely irrelevant to the class and is never exercised.

From what I can see the previous code wasn't disposing the object correctly because `MockOuterApiBinding` simply wasn't calling `MockOuterApi.Dispose()`, adding this dispose pattern doesn't have anything to do with that.